website/docs/Installation/linux-install.md

```diff
@@ -1,64 +1,131 @@
 ---
 sidebar_position: 5
+title: Linux
+description: You can install Podman Desktop on Linux from Flathub, a Flatpak bundle, or a ZIP archive.
+tags: [podman-desktop, installing, linux, flathub, flatpak]
+keywords: [podman desktop, podman, containers, installing, installation, linux, flathub, flatpak]
 ---
 
-# Linux
+# Installing Podman Desktop on Linux
 
-Check out the [Downloads](/downloads/linux) section of this website to download either the [.flatpak](https://flatpak.org/setup/Fedora) file or the zip file depending on your preference.
+You can install Podman Desktop from:
 
-To use Podman Desktop, the latest version of Podman is required. Podman is a rootless, daemonless container engine. Read more about it [here](https://podman.io/whatis.html).
+* [Flathub](#flathub)
+* [A Flatpak bundle](#flatpak-bundle)
+* [An archive](#archive)
 
-## Installing Podman Desktop
+## Installing Podman Desktop from Flathub {#flathub}
 
-### 1. Using [FlatHub](https://flathub.org/apps/details/io.podman_desktop.PodmanDesktop)
+Consider installing the Podman Desktop from Flathub to have:
 
-With FlatHub, you can install Podman Desktop with just a single command.
+* One command installation
+* Package updates
 
-> Follow the [setup guide](https://flatpak.org/setup/) before installing Podman Desktop
+#### Prerequisites
 
-```sh
-flatpak install --user flathub io.podman_desktop.PodmanDesktop
-```
-After installation, to open Podman Desktop, use the following command:
+* [Flatpak](https://flatpak.org/setup/)
+* [Podman](https://podman.io/whatis.html) stable version
 
-```sh
-flatpak run io.podman_desktop.PodmanDesktop
-```
+#### Procedure
 
-### 2. Using [Flatpak](https://flatpak.org/setup/) application from [Downloads](/downloads/linux)
+1. Add the Flathub repository:
```

Comment:
I'm wondering if it's part of the procedure or if it's a pre-requisites (in case you already have flathub available) ?